_data/guides-latest.yaml

```diff
@@ -231,9 +231,6 @@ categories:
       - title: Using Apache Kafka Streams
         url: /guides/kafka-streams
         description: This guide demonstrates how your Quarkus application can utilize the Apache Kafka Streams API to implement stream processing applications based on Apache Kafka.
-      - title: Using Apache Kafka with Schema Registry and Avro
-        url: /guides/kafka-schema-registry-avro
```

Comment:
Why did you remove this one? It's there: https://quarkus.io/guides/kafka-schema-registry-avro .

Reply:
It is a duplicate of https://github.com/quarkusio/quarkusio.github.io/pull/1062/files/533c27771da3a26f25655e34d9f1d2e3cc60c856#diff-2bcdb53c40e455971cbd6ff17e847dba549f22114c39f01dd7afbf184c2d7747L217

Reply:
@gsmet is it ok for you that I removed the duplicate?